bin/install-wp-tests.sh

```diff
@@ -69,7 +69,7 @@ install_wp() {
 
 install_pressbooks_book() {
 	if [ -d $WP_CORE_DIR/wp-content/themes/pressbooks-book ]; then
-		cd $WP_CORE_DIR/wp-content/themes/pressbooks-book && git pull
+		cd $WP_CORE_DIR/wp-content/themes/pressbooks-book && git pull origin dev
```

Comment:
After:
We now get:
```bash
There is no tracking information for the current branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

If you wish to set tracking information for this branch you can do so with:

    git branch --set-upstream-to=<remote>/<branch> fix/scssphp-deprecated-directives
```
when installing tests.
It will fix the issue.